lib/WP_Auth0_Routes.php

```diff
@@ -38,7 +38,7 @@ public function init() {
 		add_action( 'parse_request', array( $this, 'custom_requests' ) );
 	}
 
-	public function setup_rewrites( $force_ws = false ) {
```

Comment:
Allowed in PHP. Parameter was not used and anyone calling this with a parameter will not get an error.